Scripts/RemediationScripts/AutoRemediation.ps1

```diff
@@ -46,28 +45,39 @@ function StartRemediation($timestamp)
             foreach($resource in $control.FailedResourceList)
             {
                 $controlsTable+= [PSCustomObject]@{
-                    'Control Id' = if($count -eq 0) {$control.ControlId} Else {""};
-                    'Number Of Failing Resources' = if($count -eq 0) {$control.FailedResourceList.Length} Else {""};
-                    'Failing Resources' = $resource.ResourceName;
+                    'ControlId' = if($count -eq 0) {$control.ControlId} Else {""};
+                    'NumberOfFailingResources' = if($count -eq 0) {$control.FailedResourceList.Length} Else {""};
+                    'FailingResources' = $resource.ResourceName;
                 }
                 $count+=1
             }
         }
+        $colsProperty = @{Expression={$_.ControlId};Label="Control Id";Width=60;Alignment="left"},
+                        @{Expression={$_.NumberOfFailingResources};Label="Number Of Failing Resources";Width=30;Alignment="center"},
+                        @{Expression={$_.FailingResources};Label="Failing Resources";Width=30;Alignment="left"}
+
         Write-Host "Failing Controls Summary" -ForegroundColor $([Constants]::MessageType.Update)
-        $controlsTable | Format-Table 
+        $controlsTable | Format-Table -Property $colsProperty -Wrap
 
         #User input for whether to continue with current remediation or noty
         Write-Host $([Constants]::SingleDashLine)
-        $startRemediation = Read-Host -Prompt "Do you want to continue remediation? (Y/N)";
+        $startRemediation = Read-Host -Prompt "Do you want to continue remediation? (Y|N)";
 
         if(($startRemediation -eq 'Y') -or ($startRemediation -eq 'y'))
         {
-            Write-Host "User has provided consent to continue the remediation" -ForegroundColor $([Constants]::MessageType.Info)
-            $remediationLevel = Read-Host -Prompt "At which level you want to perform remediation?
-[1] Subscription: All controls will be remediated in a single flow.
-[2] Control: Confirmation will be asked before remediating each control.
+            Write-Host "User has provided consent to continue the remediation" -ForegroundColor $([Constants]::MessageType.Update)
+            Write-Host $([Constants]::SingleDashLine)
+#             $remediationLevel = Read-Host -Prompt "At which level you want to perform remediation?
+# [1] Subscription: All controls will be remediated in a single flow.
+# [2] Control: Confirmation will be asked before remediating each control.
+# Press any other key to skip the remediation of current subscription
+# Enter the choice (1|2)";
+            $remediationLevel = Read-Host -Prompt "There are two ways in all the failing resources for this subscription could be remediated.
```

Comment:
Please have a look of following message looks good.


You can choose one of the following mode to remediate non-compliant resources: 
[1] Remediate all the failing resources in a single go for all controls.
[2] Remediate failing resources control wise, confirmation will be needed before remediating failing resources against each control.
Press any other key to skip the remediation of resources in current subscription.
Enter the choice (1|2)